balloon/src/main/java/com/skydoves/balloon/Balloon.kt

```diff
@@ -817,6 +827,9 @@ class Balloon(
     /** sets the show times. */
     fun setShowTime(value: Int): Builder = apply { this.showTimes = value }
 
+    /** sets flag for using rtl layout */
+    fun setRtlLayout(value: Boolean): Builder = apply { this.isRtl = value }
```

Comment:
Same. `setLayout` functionality is getting a `LayoutResource` as a parameter.
Maybe some developers would be confused with `setLayout`.